public/index.html

```diff
@@ -26,7 +26,7 @@
     -->
     <title>React App</title>
   </head>
-  <body>
+  <body style="margin: 0">
```

Comment:
create an index.css next to top root index.js, import it and set the body margin there please! synch with Andrei he did the same.

Reply:
Done